sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -688,7 +688,7 @@ def __init__(self, state_cache, credentials=None):
 
   def create_state_handler(self, api_service_descriptor):
     # type: (endpoints_pb2.ApiServiceDescriptor) -> CachingStateHandler
-    if api_service_descriptor == endpoints_pb2.ApiServiceDescriptor():
+    if not api_service_descriptor:
```

Comment:
What about making this (either in this change or a follow-up):
`if not api_service_descriptor or api_service_descriptor == endpoints_pb2.ApiServiceDescriptor():`

Reply:
I'll do that in a follow-up so it's easier to test.